sklearn/ensemble/_base.py

```diff
@@ -33,7 +34,8 @@ def _parallel_fit_estimator(estimator, X, y, sample_weight=None):
                 ) from exc
             raise
     else:
-        estimator.fit(X, y)
+        with _print_elapsed_time(message_clsname, message):
```

Comment:
This should also include the case where sample_weight is provided. Would you like to open a new PR to fix this, @cmarmo?

Reply:
Ok, I'm on it.